cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -116,22 +122,44 @@ ON (r.propertyName_1,
 ----
 | Create a composite index on relationships.
 
+Index provider and configuration can be specified using the `OPTIONS` clause.
+
+| [source, cypher, role=noplay]
+----
+CREATE LOOKUP INDEX [index_name] [IF NOT EXISTS]
+FOR (n)
+ON EACH labels(n)
+----
+| Create a lookup index on nodes.
+
+Lookup indexes do not support any `OPTIONS` values.
+
+| [source, cypher, role=noplay]
+----
+CREATE LOOKUP INDEX [index_name] [IF NOT EXISTS]
+FOR ()-[r]-()
+ON [EACH] type(r)
+----
+| Create a lookup index on relationships.
+
+Lookup indexes do not support any `OPTIONS` values.
+
 | [source, cypher, role=noplay]
 ----
 DROP INDEX index_name [IF EXISTS]
 ----
-| Drop an index, either a node or a relationship index.
+| Drop an index, either a node, a relationship or a lookup index.
```

Comment:
```suggestion
| Drop an index, either a node, a relationship, or a lookup index.
```

Reply:
I were so good at remembering those for a while, apparently I'm back on missing them :P